### PR TITLE
Add yabai icon

### LINKED ISSUE #10968: Request: yabai
### Brand Name

yabai

### Website

https://github.com/koekeishiya/yabai

### Popularity Metric

22k ⭐ on GitHub

### Official Resources for Icon and Color

svg: https://raw.githubusercontent.com/koekeishiya/yabai/master/assets/icon/icon.svg

### Additional Comments

 A tiling window manager for macOS based on binary space partitioning

### PR DESCRIPTION
**Pull request by anhthang**
Yabai icon

**Issue:** closes #10968 

**Popularity metric:**

22k ⭐ on GitHub

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

Hex value is from original svg here: https://github.com/koekeishiya/yabai/blob/86c759186d7c588dfd8c6a84aac1ebc4730e988e/assets/icon/icon.svg

### CODE REVIEW

File: icons/yabai.svg
Comment (uncenter) [inline icons/yabai.svg:1]: Screenshot 2024-05-23 at 09 09 44 (Inkscape)

Seeing some slight differences here when comparing to the original linked in the `source`. You might need to up the precision here to `4` or higher.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: icons/yabai.svg
Comment (anhthang) [reply icons/yabai.svg:1]: Hi @uncenter, there is some small issues with the original svg, so I just re-drawn to fix it. Is it ok to go, or I need to make it same as the original one?

Comment (uncenter) [reply icons/yabai.svg:1]: What issues did you encounter? We'd prefer if you use the source SVG to start unless the last resort is redrawing.

Comment (anhthang) [reply icons/yabai.svg:1]: > What issues did you encounter? We'd prefer if you use the source SVG to start unless the last resort is redrawing.

The green & `y` path has a very small space and the combination of green, yellow, red paths is not a perfect square.
I think it's why we see some slight differences when comparing

Comment (uncenter) [reply icons/yabai.svg:1]: The combination doesn't need to be a perfect square. The larger dimension should be `24` and the aspect ratio should be preserved (at least two sides touching).

Comment (uncenter) [reply icons/yabai.svg:1]: Is it exactly 24x24 in your editor? Try increasing the precision when optimizing maybe.

File: _data/simple-icons.json
Comment (anhthang) [reply _data/simple-icons.json:17473]: that's color for yabai/y text from banner/icon

Comment (uncenter) [reply _data/simple-icons.json:17473]: Ah right. That works for me then.

**Review by uncenter — APPROVED** [review]

**Comment by anhthang** [comment]
@uncenter Are there any additional changes needed for this pull request to be merged?

**Comment by uncenter** [comment]
> @uncenter Are there any additional changes needed for this pull request to be merged?

Nope, just waiting on another reviewer.

**Comment by uncenter** [comment]
No need to update this branch, it just pings us and we'll have to update it again before merging anyway.

File: icons/yabai.svg
Comment (adamrusted) [inline icons/yabai.svg:1]: I see what you mean with having to redraw bits of this @anhthang - the joins not quite lining up is bothering me too. Perhaps worth reaching out to them to tidy up the icon, or adjusting that yourself?
image

Comment (anhthang) [reply icons/yabai.svg:1]: @adamrusted i have redrawn the green path a bit to fix the issue. hope its ok now. can you check again?

**Review by adamrusted — CHANGES_REQUESTED** [review]
Weirdly, whatever you did added _more_ differences!
image

See the lines between the `y` shape and the diamond. Ideally they should be removed so it's just one solid path.

**Comment by anhthang** [comment]
@adamrusted I slightly modified the green path to make it easier to combine it seamlessly with the Y shape. There shouldn't be any significant changes compared to the previous version. Could you please review it again to see if the issue is resolved?

File: icons/yabai.svg
Comment (adamrusted) [inline icons/yabai.svg:1]: I'm now seeing a slight difference in the 'tail' of the 'y' - see below.
Would you be able to see what might be causing that? Potentially just a precision thing.
image

Comment (anhthang) [reply icons/yabai.svg:1]: fixed, it looks great from my end now. hope this is final 😅

**Review by adamrusted — APPROVED** [review]
YES! We've gotten there! Thanks for your patience @anhthang!

**Pull request merged**
